pandas/core/reshape/tile.py

```diff
@@ -294,11 +296,14 @@ def qcut(
     duplicates: str = "raise",
 ):
     """
-    Quantile-based discretization function.
-
-    Discretize variable into equal-sized buckets based on rank or based
-    on sample quantiles. For example 1000 values for 10 quantiles would
-    produce a Categorical object indicating quantile membership for each data point.
+    Transforms a variable with a large number of possibly continuous values 
+    into a variable with a smaller number of quantile-based intervals, or bins.
+    
+    Use `qcut` when you need to place values into equal (by 
+    the number of placed values) of unequal quantile-based bins. For example, 
```

Comment:
`place values into equal-quantity or predefined quantile-based bins, all of likely unequal-width.`